app/models/beta_pages.rb

```diff
@@ -1,7 +1,16 @@
 class BetaPages
   def self.all
     [
-      'pipelines/cluster-queue-metrics'
+      'agent/clusters',
```

Comment:
Should the clusters API be marked "beta".

Clusters went GA recently (see #2642 for more context).